### PR TITLE
RELATED: RAIL-2819 add Dashboard item components 

### PR DESCRIPTION
**Pull request by no23reason**
Adds visual components for the dashboard items along with their CSS.
The clientWidth of the DashboardItemContent is not used for now, but it will be necessary once we have the whole KPI component migrated and when we use the InsightView more directly (it may need a new prop for this to pass it to the PlugVis).
Also, KpiView and friends had the title prop removed as it is handled higher up the tree now.

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [x] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [ ] `check-extended` passes
-   [x] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1791/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3393/ : FAILURE in 5m 29s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1792/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3394/ : SUCCESS in 14m 29s

File: libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardItem/DashboardItemVisualization.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+     * Class name applied to the main content.
+     */
+    contentClassName?: string;
+    /**
+     * Class name applied to the visualization container.
+     */
+    visualizationClassName?: string;
+}
+
+export const DashboardItemVisualization: React.FC<IDashboardItemVisualizationProps> = ({
```
Comment (kandl) [inline libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardItem/DashboardItemVisualization.tsx:43]: This component looks very similar as DashboardItemKpi (only 1 css class is different if I'm correct - we should probably "merge" them and speficy this css class/flag from the parent component?)

**Review by kandl — APPROVED** [review]
LGTM

File: libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardItem/DashboardItemVisualization.tsx (hunk 1 above)
Comment (no23reason) [reply libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardItem/DashboardItemVisualization.tsx]: Simplified, thanks 👍

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/574/ : FAILURE in 2m 03s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/835/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3402/ : SUCCESS in 10m 49s

**Pull request merged**